dist/s_clang_scan

```diff
@@ -39,26 +39,21 @@ rm -rf ${t_build} && mkdir ${t_build}
 cd ${t_build}
 args="-o clangScanBuildReports"
 args="$args --use-cc=$compiler"
-args="$args -disable-checker core.NullDereference"
 $scan $args cmake CMAKE_C_FLAGS=-g \
-    -DENABLE_STATIC=1 -DENABLE_SHARED=0 -DHAVE_DIAGNOSTIC=1 -DENABLE_STRICT=1 ../.> /dev/null
-$scan $args make -j 8 wt 1>../$t_out 2>../$t_err
+    -DENABLE_STATIC=1 -DENABLE_SHARED=0 -DHAVE_DIAGNOSTIC=1 -DENABLE_STRICT=1 -G Ninja ../.> /dev/null
+$scan $args ninja wt > ../$t_out 2>&1
 cd ..
 
-grep 'No bugs found' $t_out > /dev/null && exit 0
```

Comment:
Removed this as warnings are not considered bugs. We only rely on the `diff` done a few lines down.

Reply:
Does this mean we don't expect 'No bugs found' wording anymore, or the diff later covers this check?

Reply:
The issue is that we may have warnings and still have this log "No bugs found". In my opinion, we should check every warning instead of relying on that sentence that is meaningless now. So yes, the diff is covering what we need to check.

Reply:
Got it, thanks